.codespellrc

```diff
@@ -8,6 +8,7 @@ exclude-file = .codespell-ignore-lines
 # Ignore complete files (e.g. legal text or other immutable material).
 skip =
   LICENSE,
+  CODEOWNERS,
```

Comment:
```suggestion
  */CODEOWNERS,
```

Reply:
Huh, this worked. For some reason it did not work with `.github/CODEOWNERS` though. Thank you!

Reply:
or better: `*/.github/CODEOWNERS`